.mergify.yml

```diff
@@ -417,3 +417,17 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
+  - name: backport patches to 8.18 branch
+    conditions:
+      - merged
+      - base=8.x
```

Comment:
This won't be the case, as soon as https://github.com/elastic/apm-server/pull/14392 is merged and backported to `8.x`.